tutorials/simulators/OpenFOAM.md

```diff
@@ -358,40 +218,43 @@ much more.
     task.print_summary()
     ```
 
-### Step 4: Enhancing Performance with MPI Cluster
+### Conclusion
 
-As you have experienced, simulations can take a long, long time. To further
-reduce runtime we can change our machine configuration to a MPI cluster
-with two machines:
+Running your simulation on a high-performance cloud machine can significantly
```

Comment:
I don't think you can take this conclusion because we don't show a comparison.
I think we should run the simulation (of a % of it) end to end. @Pbarbosa92 : let's make sure we actually run one simulation end to end, even if it takes a couple of hours (but not a couple of days, ok?). I would like to show the result of print_summary(). Then we can run the simulation again using another machine (better or worse) so we can show a different result and take conclusion.

Reply:
Well, I can't actually run this simulation once I change the simulation time. Reducing the simulation time results in crashes. I'll try a couple of other things, like increasing the time step but i belive that will also increase instability.

Reply:
ok! How long does the original version take to run?

Reply:
Almost 2 days on a 360 machine.

Reply:
That's too much. Let's try to cut that 10x.